files/justfiles/utilities.just

```diff
@@ -200,13 +201,35 @@ install-steam:
             ;;
         2)
             echo "Distrobox method selected."
+            if [ -z "$(semodule -l | grep -q 'harden_container_userns')" ] ; then
+                echo "Distrobox requires that secureblue's harden_container_userns module be disabled"
+                echo "Please run 'ujust toggle-container-domain-userns-creation'"
+                container_userns_now=""
+                read -p "Would you like to run 'ujust toggle-container-domain-userns-creation' now? [Y/n] " container_userns_now
+                container_userns_now=${container_userns_now:-y}
+                if [[ "$container_userns_now" == [Yy]* ]]; then
+                    echo "Running 'ujust toggle-container-domain-userns-creation'"
+                    ujust toggle-container-domain-userns-creation
+                fi
+            fi
             echo "Creating bazzite-arch distrobox."
             distrobox-create --unshare-netns --nvidia --image ghcr.io/ublue-os/bazzite-arch --name bazzite-arch -Y
             echo "Exporting Steam from bazzite-arch distrobox."
             distrobox-enter -n bazzite-arch -- distrobox-export --app steam
             ;;
         3)
             echo "Native layer method selected."
+            if [ -z "$(semodule -l | grep -q 'harden_userns')" ] ; then
```

Comment:
`semodule -l` requires root, im thinking of an alternative here since run0 when used directly for this command doesn't provide output, and if we run0 the whole ujust, the distrobox will get installed for the root user